File: student_management_app/lib/models/course.dart
class Course {
  final int id;
  final String name;
  final String description;
  final int durationMonths;
  final bool isActive;

  Course({
    required this.id,
    required this.name,
    required this.description,
    required this.durationMonths,
    required this.isActive,
  });

  factory Course.fromJson(Map<String, dynamic> json) {
    return Course(
      id: json['id'] ?? 0,
      name: json['name'] ?? '',
      description: json['description'] ?? '',
      durationMonths: json['durationMonths'] ?? 0,
      isActive: json['isActive'] ?? false,
    );
  }

  Map<String, dynamic> toJson() {
    return {
      'id': id,
      'name': name,
      'description': description,
      'durationMonths': durationMonths,
      'isActive': isActive,
    };
  }
}

File: StudentManagement.API/Models/Course.cs
namespace StudentManagement.API.Models
{
    public class Course
    {
        public int Id { get; set; }

        public string Name { get; set; } = string.Empty;

        public string Description { get; set; } = string.Empty;

        public int DurationMonths { get; set; }

        public bool IsActive { get; set; } = true;

        // ============================================================
        // ORGANIZATION RELATIONSHIP
        // ============================================================

        public int OrganizationId { get; set; }

        public Organization? Organization { get; set; }

        // ============================================================
        // NEW MULTIPLE STUDENTS RELATIONSHIP
        // ============================================================

        public ICollection<StudentCourse> StudentCourses { get; set; }
            = new List<StudentCourse>();
    }
}